ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -111,6 +111,18 @@
             }
         }
 
+        updateValidity(validity) {
+            const valid = validity.valid;
```

Comment:
I will re-write it as:
` const valid = validity.valid ? validity.valid : false;
            let widgets = this.widget;
            widgets.forEach(widget => {
                     widget.setAttribute(FormView.Constants.ARIA_INVALID, !valid);
            });`

Reply:
Implemented!